src/components/tracked-controls-webxr.js

```diff
@@ -62,17 +62,26 @@ module.exports.Component = registerComponent('tracked-controls-webxr', {
   },
 
   emitButtonDownEvent: function (evt) {
-    if (!this.controller || evt.inputSource.handedness !== this.data.hand) { return; }
+    if (!this.controller) { return; }
     if (this.controller.gamepad) { return; }
+    if (evt.inputSource.handedness !== 'none' &&
+        evt.inputSource.handedness !== this.data.hand) {
+      return;
+    }
+
     this.selectEventDetails.state.pressed = true;
     this.el.emit('buttondown', this.selectEventDetails);
     this.el.emit('buttonchanged', this.selectEventDetails);
     this.el.emit('triggerdown');
   },
 
   emitButtonUpEvent: function (evt) {
-    if (!this.controller || evt.inputSource.handedness !== this.data.hand) { return; }
+    if (!this.controller) { return; }
```

Comment:
Why is this needed? What's particular about magic leap with respect to other controllers?

Reply:
Is magicleap controller always returning `none` as handness or is it user configurable too like Oculus Go?

Reply:
regarding this, we currently do not specify a handedness for our controllers (although, in theory it is possible for us to recognize which hand is currently holding the controller, and set it at runtime; however, we currently do not have that planned as a feature for our controllers).

Since Chromium Code enumerates a NONE state, and the webxr reference does mention a value of "" is possible, it is my personal believe that there should be handling for such a scenario where it is neither left nor right. (I'm actually not sure why the reference page lists an empty string, since it seems like it's going to set to none).